test/lib/ansible_test/_internal/units/__init__.py

```diff
@@ -99,6 +99,7 @@ def command_units(args):
             '-p', 'no:cacheprovider',
             '-c', os.path.join(ANSIBLE_TEST_DATA_ROOT, 'pytest.ini'),
             '--junit-xml', os.path.join(data_context().results, 'junit', 'python%s-units.xml' % version),
+            '--durations=25',
```

Comment:
My initial response to this was to think it's a good idea. However, that was thinking of its usage in the context of ansible/ansible. I'm less certain we should do this for testing collections.

Reply:
Ok, so just toggle off of whether this is a collection or not?

Reply:
Yes, for now, lets just do it for non-collection testing.